src/main/java/spoon/reflect/factory/ExecutableFactory.java

```diff
@@ -79,7 +79,11 @@ public CtAnonymousExecutable createAnonymous(CtClass<?> target, CtBlock<Void> bo
 	public <T> CtParameterReference<T> createParameterReference(CtParameter<T> parameter) {
 		CtParameterReference<T> ref = factory.Core().createParameterReference();
 		ref.setSimpleName(parameter.getSimpleName());
-		ref.setType(parameter.getType());
+		if (parameter.getType() == null) {
+			ref.setType(parameter.getType());
+		} else {
+			ref.setType(parameter.getType().clone());
+		}
```

Comment:
This solution works but it hides an implicit difference between ordinary parameters and parameters in lambda expressions. Is it better to create a CtLambdaParameter class?